Description: Upstream changes introduced in version 6.24.3-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 peak-linux-driver (6.24.3-1) unstable; urgency=low
 .
   * Integrated upstream patch.
 .
 The person named in the Author field signed this changelog entry.
Author: Markus Becker <mab@comnets.uni-bremen.de>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ peak-linux-driver-6.24.3/peak-linux-driver-6.24.3.patch
@@ -0,0 +1,257 @@
+--- driver/release_notes.txt	2011-01-13 10:46:05.000000000 +0100
++++ driver/release_notes.txt	2011-02-11 16:13:37.000000000 +0100
+@@ -1,2 +1,8 @@
+-2011-01-12 - v6.24 - content:
+-- Handle PCMCIA cards in v2.6.35 kernel
++2011-01-12 - 6.24 - content:
++- Handle PCMCIA cards in 2.6.35 kernel
++
++2011-01-18 - 6.24.2 - content:
++- Support 2.6.37 kernel
++
++2011-02-11 - 6.24.3 - content:
++- Support 2.6.37 kernel and PCMCIA
+--- driver/src/pcan_common.h	2011-01-13 10:46:05.000000000 +0100
++++ driver/src/pcan_common.h	2011-02-11 16:16:26.000000000 +0100
+@@ -185,7 +185,7 @@
+ #define CURRENT_RELEASE "Release_20110113_n"  // $name$
+ #define PCAN_VERSION_MAJOR             6
+ #define PCAN_VERSION_MINOR             24
+-#define PCAN_VERSION_SUBMINOR          0
++#define PCAN_VERSION_SUBMINOR          3
+ #define CURRENT_VERSIONSTRING          __stringify(PCAN_VERSION_MAJOR)"."__stringify(PCAN_VERSION_MINOR)"."__stringify(PCAN_VERSION_SUBMINOR)
+ 
+ 
+--- driver/src/pcan_fops_linux.c	2010-02-14 23:38:55.000000000 +0100
++++ driver/src/pcan_fops_linux.c	2011-02-11 16:12:44.000000000 +0100
+@@ -422,7 +422,11 @@
+ 
+ //----------------------------------------------------------------------------
+ // is called at user ioctl() call
++#if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,36)
+ int pcan_ioctl(struct inode *inode, struct file *filep, unsigned int cmd, unsigned long arg)
++#else
++long pcan_ioctl(struct file *filep, unsigned int cmd, unsigned long arg)
++#endif
+ {
+   int err;
+   struct fileobj *fobj = (struct fileobj *)filep->private_data;
+@@ -741,7 +745,11 @@
+   release:    pcan_release,
+   read:       pcan_read,
+   write:      pcan_write,
++#if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,36)
+   ioctl:      pcan_ioctl,
++#else
++  unlocked_ioctl: pcan_ioctl,
++#endif
+   poll:       pcan_poll,
+ };
+ 
+--- driver/src/pcan_main.h	2010-09-26 22:44:05.000000000 +0200
++++ driver/src/pcan_main.h	2011-02-11 16:12:44.000000000 +0100
+@@ -68,8 +68,12 @@
+ #endif
+ 
+ #ifdef PCCARD_SUPPORT
++#if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,37)
++#if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,36)
+ #include <pcmcia/cs_types.h>
++#endif
+ #include <pcmcia/cs.h>
++#endif
+ #include <pcmcia/cistpl.h>
+ #include <pcmcia/ds.h>
+ #endif
+--- driver/src/pcan_pccard-2.6.17.c	2011-01-12 14:15:30.000000000 +0100
++++ driver/src/pcan_pccard-2.6.17.c	2011-02-11 16:12:44.000000000 +0100
+@@ -54,6 +54,7 @@
+ 
+ //****************************************************************************
+ // helper function to get retrieved configuraton
++#if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,37)
+ static int pcan_pccard_conf_check(struct pcmcia_device *link, cistpl_cftable_entry_t *cfg,
+                                 cistpl_cftable_entry_t *dflt, unsigned int vcc, void *private)
+ {
+@@ -63,17 +64,45 @@
+   {
+     cistpl_io_t  *io  = (cfg->io.nwin > 0) ? &cfg->io  : &dflt->io;
+ 
++#if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,36)
+     link->io.BasePort1     = io->win[0].base;
+     link->io.NumPorts1     = io->win[0].len;
+     link->io.Attributes1   = IO_DATA_PATH_WIDTH_8; // only this kind of access is yet supported
+     link->io.IOAddrLines   = io->flags &  CISTPL_IO_LINES_MASK;
+ 
+     if (pcmcia_request_io(link, &link->io) == 0)
++#else
++    struct resource *pr = link->resource[0];
++
++    pr->start = io->win[0].base;
++    pr->end = io->win[0].len;
++    pr->name = dev_name(&link->dev);
++    pr->flags |= IO_DATA_PATH_WIDTH_8; // only this kind of access is yet supported
++    link->io_lines = io->flags &  CISTPL_IO_LINES_MASK;
++
++    if (pcmcia_request_io(link) == 0)
++#endif
++
+       return 0;
+   }
+ 
+   return -ENODEV;
+ }
++#else
++static int pcan_pccard_conf_check(struct pcmcia_device *link, void *private)
++{
++   struct resource *pr = link->resource[0];
++
++   DPRINTK(KERN_DEBUG "%s: %s()\n", DEVICE_NAME, __FUNCTION__);
++
++   pr->name = dev_name(&link->dev);
++   pr->flags &= ~IO_DATA_PATH_WIDTH;
++   pr->flags |= IO_DATA_PATH_WIDTH_8; // only this kind of access is yet supported
++   link->io_lines = 10;
++
++   return pcmcia_request_io(link); 
++}
++#endif
+ 
+ 
+ #if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,28)
+@@ -208,6 +237,10 @@
+ 
+   DPRINTK(KERN_DEBUG "%s: pccard_plugin(0x%p)\n", DEVICE_NAME, link);
+ 
++#if LINUX_VERSION_CODE >= KERNEL_VERSION(2,6,37)
++  link->config_flags |= CONF_AUTO_SET_IO;
++#endif
++
+   if ((last_ret = pcmcia_loop_config(link, pcan_pccard_conf_check, NULL)))
+   {
+     printk(KERN_WARNING "%s: pcmcia_loop_config() = %d!", DEVICE_NAME, last_ret);
+@@ -239,35 +272,45 @@
+   /* (this the job of pccard_create_all_devices() call below) */
+ #endif
+ 
++#if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,37)
+   link->conf.Attributes = CONF_ENABLE_IRQ;
+   link->conf.IntType    = INT_MEMORY_AND_IO;
+   link->conf.Present    = PRESENT_OPTION | PRESENT_STATUS;
+   if ((last_ret = pcmcia_request_configuration(link, &link->conf)))
+   {
+     printk(KERN_WARNING "%s: pcmcia_request_configuration() = %d!", DEVICE_NAME, last_ret);
++#else
++  link->config_flags |= CONF_ENABLE_IRQ;
++  link->config_regs = PRESENT_OPTION;
++  if ((last_ret = pcmcia_enable_device(link)))
++  {
++     printk(KERN_WARNING "%s: pcmcia_enable_device() = %d!", DEVICE_NAME, last_ret);
++#endif
+     goto fail;
+   }
+ 
+ #if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,35)
+-  DPRINTK(KERN_DEBUG "%s: pccard found: base1=0x%04x, size=%d, irq=%d\n", DEVICE_NAME, link->io.BasePort1, link->io.NumPorts1, link->irq.AssignedIRQ);
+-
+   // init (cardmgr) devices associated with that card (is that necessary?)
+   card->node.major = pcan_drv.nMajor;
+   card->node.minor = PCCARD_MINOR_BASE;
+   strcpy(card->node.dev_name, DEVICE_NAME);
+   link->dev_node = &card->node;
++  card->commonIrq = link->irq.AssignedIRQ;
+ #else
+-  DPRINTK(KERN_DEBUG "%s: pccard found: base1=0x%04x, size=%d, irq=%d\n", DEVICE_NAME, link->io.BasePort1, link->io.NumPorts1, link->irq);
++  card->commonIrq = link->irq;
+ #endif
+ 
+-  // create device descriptors associated with the card - get relevant parts to get independent from dev_link_t
++#if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,36)
+   card->basePort  = link->io.BasePort1;
+-#if LINUX_VERSION_CODE >= KERNEL_VERSION(2,6,35)
+-  card->commonIrq = link->irq;
++  card->numPort = link->io.NumPorts1;
+ #else
+-  card->commonIrq = link->irq.AssignedIRQ;
++  card->basePort = link->resource[0]->start;
++  card->numPort = link->resource[0]->end;
+ #endif
+ 
++  DPRINTK(KERN_DEBUG "%s: pccard found: base1=0x%04x, size=%d, irq=%d\n", DEVICE_NAME, card->basePort, card->numPort, card->commonIrq);
++
++  // create device descriptors associated with the card - get relevant parts to get independent from dev_link_t
+   last_ret = pccard_create_all_devices(card);
+   if (last_ret)
+     goto fail;
+--- driver/src/pcan_pccard.c	2011-01-12 14:15:30.000000000 +0100
++++ driver/src/pcan_pccard.c	2011-02-11 16:15:21.000000000 +0100
+@@ -43,8 +43,12 @@
+ #include <linux/string.h>
+ #include <linux/ioport.h>
+ 
++#if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,37)
++#if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,36)
+ #include <pcmcia/cs_types.h>
++#endif
+ #include <pcmcia/cs.h>
++#endif
+ #include <pcmcia/cistpl.h>
+ #include <pcmcia/ds.h>
+ #include <pcmcia/cisreg.h>
+@@ -119,6 +123,11 @@
+   pcan_drv.pccarddrv.remove     = pccard_detach;
+   pcan_drv.pccarddrv.suspend    = pccard_suspend;
+   pcan_drv.pccarddrv.resume     = pccard_resume;
++#if LINUX_VERSION_CODE >= KERNEL_VERSION(2,6,37)
++  /* this version sets pccarddrv->drv.name with pccarddrv.name => */
++  /* it must be a valid pointer too */
++  pcan_drv.pccarddrv.name       = DEVICE_NAME;
++#endif
+   #endif
+ 
+   return pcmcia_register_driver(&pcan_drv.pccarddrv);
+--- driver/src/pcan_pccard_kernel.h	2011-01-12 14:15:30.000000000 +0100
++++ driver/src/pcan_pccard_kernel.h	2011-02-11 16:12:44.000000000 +0100
+@@ -44,8 +44,12 @@
+ #include <linux/timer.h>         // to support activity scan
+ 
+ #include <linux/types.h>
++#if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,37)
++#if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,36)
+ #include <pcmcia/cs_types.h>
++#endif
+ #include <pcmcia/cs.h>
++#endif
+ #include <pcmcia/cistpl.h>
+ #include <pcmcia/ds.h>
+ 
+@@ -65,6 +69,7 @@
+   dev_link_t link;
+ #endif
+   u_int      basePort;                       // base of io area for all channels
++  u_int      numPort;
+   u_int      commonIrq;                      // irq for all channels
+   u_int      commonPort;                     // channels commonly used port
+   struct     pcandev *dev[PCCARD_CHANNELS];  // point to associated channels
+--- driver/src/pcan_usb_kernel.c	2010-02-14 23:38:55.000000000 +0100
++++ driver/src/pcan_usb_kernel.c	2011-02-11 16:12:44.000000000 +0100
+@@ -223,7 +223,8 @@
+   int nResult = 0;
+   register purb_t pt;
+ 
+-  // DPRINTK(KERN_DEBUG "%s: pcan_set_parameter()\n", DEVICE_NAME);
++  DPRINTK(KERN_DEBUG "%s: %s(): ->EP#%02Xh\n", DEVICE_NAME, __FUNCTION__,
++dev->port.usb.Endpoint[1].ucNumber);
+ 
+   // don't do anything with non-existent hardware
+   if (!dev->ucPhysicallyInstalled)
+@@ -295,7 +296,8 @@
+   register purb_t pt;
+   USB_PORT *u = &dev->port.usb;
+ 
+-  DPRINTK(KERN_DEBUG "%s: pcan_hw_getcontrol_urb(%d, %d)\n", DEVICE_NAME, function, number);
++  DPRINTK(KERN_DEBUG "%s: %s(): <-EP#%02Xh\n", DEVICE_NAME, __FUNCTION__,
++dev->port.usb.Endpoint[0].ucNumber);
+ 
+   // don't do anything with non-existent hardware
+   if (!dev->ucPhysicallyInstalled)
--- /dev/null
+++ peak-linux-driver-6.24.3/driver/src/.pcan_fops.o.d
@@ -0,0 +1,420 @@
+pcan_fops.o: \
+ /home/mab/src/external/git/peak-linux-driver/driver/src/pcan_fops.c \
+ include/generated/autoconf.h \
+ /home/mab/src/external/git/peak-linux-driver/driver/src/pcan_common.h \
+ include/linux/version.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/stringify.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/module.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/list.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/types.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/types.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/types.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/int-ll64.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/bitsperlong.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/bitsperlong.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/posix_types.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/stddef.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/compiler.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/compiler-gcc.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/compiler-gcc4.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/align-section.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/posix_types.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/posix_types_32.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/poison.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/prefetch.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/processor.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/processor-flags.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/vm86.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/ptrace.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/ptrace-abi.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/segment.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/page_types.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/const.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/page_32_types.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/init.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/math_emu.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/sigcontext.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/current.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/percpu.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/kernel.h \
+ /usr/lib/gcc/i486-linux-gnu/4.4.5/include/stdarg.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/linkage.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/linkage.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/bitops.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/bitops.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/alternative.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/jump_label.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/asm.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/cpufeature.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/required-features.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/bitops/find.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/bitops/sched.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/arch_hweight.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/bitops/const_hweight.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/bitops/fls64.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/bitops/ext2-non-atomic.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/bitops/le.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/byteorder.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/byteorder/little_endian.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/swab.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/swab.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/byteorder/generic.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/bitops/minix.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/log2.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/typecheck.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/printk.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/dynamic_debug.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/bug.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/bug.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/div64.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/percpu.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/threads.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/percpu-defs.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/system.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/cmpxchg.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/cmpxchg_32.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/nops.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/irqflags.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/irqflags.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/paravirt.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/pgtable_types.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/pgtable_32_types.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/pgtable-2level_types.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/pgtable-nopud.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/pgtable-nopmd.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/paravirt_types.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/desc_defs.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/kmap_types.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/kmap_types.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/cpumask.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/bitmap.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/string.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/string.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/string_32.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/page.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/page_32.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/memory_model.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/getorder.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/msr.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/msr-index.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/ioctl.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/ioctl.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/ioctl.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/errno.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/errno.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/errno-base.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/cpumask.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/personality.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/cache.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/cache.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/math64.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/err.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/stat.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/stat.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/time.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/seqlock.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/spinlock.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/preempt.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/thread_info.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/thread_info.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/ftrace.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/atomic.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/atomic64_32.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/atomic-long.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/bottom_half.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/spinlock_types.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/spinlock_types.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/lockdep.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/rwlock_types.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/spinlock.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/rwlock.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/rwlock.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/spinlock_api_smp.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/rwlock_api_smp.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/kmod.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/gfp.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/mmzone.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/wait.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/numa.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/nodemask.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/pageblock-flags.h \
+ include/generated/bounds.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/memory_hotplug.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/notifier.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/errno.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/mutex.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/rwsem.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/rwsem.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/srcu.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/topology.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/smp.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/smp.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/mpspec.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/mpspec_def.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/x86_init.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/bootparam.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/screen_info.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/apm_bios.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/edd.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/e820.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/ioport.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/ist.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/video/edid.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/apic.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/delay.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/delay.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/pm.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/workqueue.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/timer.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/ktime.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/jiffies.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/timex.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/param.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/param.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/param.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/timex.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/tsc.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/debugobjects.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/completion.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/apicdef.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/fixmap.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/acpi.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/acpi/pdc_intel.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/numa.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/numa_32.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/mmu.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/io_apic.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/irq_vectors.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/percpu.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/pfn.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/topology.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/topology.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/mmdebug.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/elf.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/elf-em.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/elf.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/user.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/user_32.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/auxvec.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/vdso.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/desc.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/ldt.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/kobject.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/sysfs.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/kobject_ns.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/kref.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/moduleparam.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/tracepoint.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/rcupdate.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/rcutree.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/module.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/module.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/trace/events/module.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/trace/define_trace.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/slab.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/slub_def.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/kmemleak.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/trace/events/kmem.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/trace/events/gfpflags.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/fs.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/limits.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/blk_types.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/kdev_t.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/dcache.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/rculist.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/path.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/radix-tree.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/prio_tree.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/pid.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/capability.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/semaphore.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/fiemap.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/quota.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/percpu_counter.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/dqblk_xfs.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/dqblk_v1.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/dqblk_v2.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/dqblk_qtree.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/nfs_fs_i.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/nfs.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/sunrpc/msg_prot.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/inet.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/fcntl.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/fcntl.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/fcntl.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/proc_fs.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/magic.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/pci.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/pci_regs.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/mod_devicetable.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/device.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/klist.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/device.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/pm_wakeup.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/io.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/io.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/xen/xen.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/iomap.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/vmalloc.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/irqreturn.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/pci_ids.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/pci-dma.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/dmapool.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/scatterlist.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/scatterlist.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/pci.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/mm.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/rbtree.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/debug_locks.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/mm_types.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/auxvec.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/page-debug-flags.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/range.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/pgtable.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/pgtable_32.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/pgtable_32_types.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/pgtable-2level.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/pgtable.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/page-flags.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/vmstat.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/pci-dma-compat.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/dma-mapping.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/dma-attrs.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/bug.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/scatterlist.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/dma-mapping.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/kmemcheck.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/dma-debug.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/swiotlb.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/swiotlb.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/dma-coherent.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/dma-mapping-common.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/pci.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/uaccess.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/uaccess_32.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/poll.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/poll.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/poll.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/sysctl.h \
+ /home/mab/src/external/git/peak-linux-driver/driver/src/pcan_main.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/interrupt.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/irqnr.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/hardirq.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/ftrace_irq.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/hardirq.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/irq.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/irq.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/irq_regs.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/irqdesc.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/hw_irq.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/profile.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/sections.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/sections.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/hrtimer.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/trace/events/irq.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/parport.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/i2c.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/i2c-id.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/sched.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/cputime.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/cputime.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/sem.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/ipc.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/ipcbuf.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/ipcbuf.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/sembuf.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/signal.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/signal.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/signal-defs.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/siginfo.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/siginfo.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/proportions.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/seccomp.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/seccomp.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/seccomp_32.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/unistd.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/unistd.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/unistd_32.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/rtmutex.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/plist.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/resource.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/resource.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/resource.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/task_io_accounting.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/latencytop.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/cred.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/key.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/selinux.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/aio.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/aio_abi.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/uio.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/of.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/i2c-algo-bit.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/usb.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/usb/ch9.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/can.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/socket.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/socket.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/socket.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/sockios.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/asm-generic/sockios.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/sockios.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/can/error.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/if_arp.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/netdevice.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/if.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/hdlc/ioctl.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/if_ether.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/skbuff.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/net.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/random.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/ratelimit.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/textsearch.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/net/checksum.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/checksum.h \
+ /usr/src/linux-headers-2.6.37-1-common/arch/x86/include/asm/checksum_32.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/in6.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/dmaengine.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/if_packet.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/if_link.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/netlink.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/pm_qos_params.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/miscdevice.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/major.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/ethtool.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/net/net_namespace.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/net/netns/core.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/net/netns/mib.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/net/snmp.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/snmp.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/u64_stats_sync.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/net/netns/unix.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/net/netns/packet.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/net/netns/ipv4.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/net/inet_frag.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/net/netns/ipv6.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/net/dst_ops.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/net/netns/dccp.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/net/netns/x_tables.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/netfilter.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/in.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/net/flow.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/net/netns/conntrack.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/list_nulls.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/net/netns/xfrm.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/xfrm.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/seq_file_net.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/linux/seq_file.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/net/dsa.h \
+ /usr/src/linux-headers-2.6.37-1-common/include/net/dcbnl.h \
+ /home/mab/src/external/git/peak-linux-driver/driver/pcan.h \
+ /home/mab/src/external/git/peak-linux-driver/driver/src/pcan_pci.h \
+ /home/mab/src/external/git/peak-linux-driver/driver/src/pcan_isa.h \
+ /home/mab/src/external/git/peak-linux-driver/driver/src/pcan_dongle.h \
+ /home/mab/src/external/git/peak-linux-driver/driver/src/pcan_sja1000.h \
+ /home/mab/src/external/git/peak-linux-driver/driver/src/pcan_fifo.h \
+ /home/mab/src/external/git/peak-linux-driver/driver/src/pcan_fops.h \
+ /home/mab/src/external/git/peak-linux-driver/driver/src/pcan_parse.h \
+ /home/mab/src/external/git/peak-linux-driver/driver/src/pcan_usb.h \
+ /home/mab/src/external/git/peak-linux-driver/driver/src/pcan_filter.h \
+ /home/mab/src/external/git/peak-linux-driver/driver/src/pcan_fops_linux.c
